src/utils.ts

```diff
@@ -95,12 +95,7 @@ export const pivotal = (pivotalToken: string) => {
  * @param {IssuesAddLabelsParams} labelData
  */
 export const addLabels = async (client: github.GitHub, labelData: IssuesAddLabelsParams) => {
-  try {
-    await client.issues.addLabels(labelData);
-  } catch (error) {
-    core.setFailed(error.message);
-    process.exit(1);
-  }
```

Comment:
any particular reason on removing this `try...catch` block?

Reply:
the thought was that these errors are already caught in main, and since we added more specific error handling in there it was nice to have the errors bubble up and be handled in main.